addon-test-support/settings.js

```diff
@@ -5,7 +5,7 @@
 */
 class TestSupportSettings {
 
-  constructor(init = { rootElement: '#ember-testing' }) {
+  constructor(init = { rootElement: '#ember-testing > .ember-view' }) {
```

Comment:
I think we should expose this from `ember-test-helpers`.  I don't _really_ like encoding this in various places, as it has changed over time and might change again in the future.  

I'm thinking something like:

```js
import { rootElementSelector as rootElement } from 'ember-test-helpers';

class TestSupportSettings {
  constructor(init = { rootElement }) {

  }
}
```

What do y'all think?

Reply:
💯 agree!

See also https://github.com/simplabs/qunit-dom/issues/39 /cc @Turbo87

Reply:
@rwjblue what about the app (dummay) config also having the ability to set the rootElement? for some reason that was important when we originally defined the rootElement usage with the `find` helper.

Reply:
I was using this ability [here](https://github.com/kaliber5/ember-bootstrap/blob/master/tests/test-helper.js#L16) to "fix" the rootElement for my case, and I think we can still keep this feature, but just take the default from the `ember-test-helpers` export when that lands!

Reply:
@rwjblue @simonihmig what about importing the root element from config?

```
import config from '../config/environment'; // does this work? no cause it looks in addon?
const { APP: { rootElement } } = config;

class TestSupportSettings {
  constructor(init = { rootElement }) {

  }
}
```

Reply:
I think your comment is right, it won't work unless the module is in the `app` tree. You would have to dynamically look up the config from DI, afaik. Also `rootElement` in `environment.js` is just `'#ember-testing'` (for test env), while there will still be that wrapping `.ember-view`.